src/components/MonthCalendar.spec.tsx

```diff
@@ -56,23 +108,24 @@ describe('MonthCalendar', () => {
 
     it('displays the correct date on each grid square', () => {
         mountWithProviders(
-            <MonthCalendar events={ dummyEvents } month={ dummyMonth } year={ dummyYear } />,
+            <MonthCalendar events={ dummyEvents } month={ 4 } year={ 2021 } />,
```

Comment:
This is so much nicer IMO! Now I can see all I need to understand what's going on inside this single test function, that fits within my editor without scrolling. So it's less code, and easier to read! 🏆